specification/storagemover/resource-manager/readme.md

```diff
@@ -27,18 +27,36 @@ These are the global settings for the storagemover.
 ``` yaml
 openapi-type: arm
 openapi-subtype: providerHub
-tag: package-2023-10
+tag: package-2024-07
```

Comment:
>024-07 [](http://example.com/codeflow?start=14&length=6)

The first commit needs to be an exact copy of the previous api version. All new changes should only be added in the subsequent commits. 

This is explained in detail here - [https://eng.ms/docs/products/azure-developer-experience/design/api-specs/api-specs?tabs=openapi](https://eng.ms/docs/products/azure-developer-experience/design/api-specs/api-specs?tabs=openapi) 

Please remove the "ARMChangesRequested" label once you have fixed this issue.

Reply:
@razvanbadea-msft I'm unable to remove the label due to lack of permissions. I submitted a request for that, but the permissions haven't been updated yet.